kuma/wiki/templatetags/jinja_helpers.py

```diff
@@ -241,12 +241,17 @@ def wiki_url(path):
 
 @library.global_function
 @lru_cache.lru_cache()
-def include_svg(path, title=None):
-    """Embded an SVG file by path, optionally changing the title."""
+def include_svg(path, title=None, title_id=None):
```

Comment:
This is cool! It makes me worry about performance, though... I'd be happier with this on the client than on the server. But that is theoretical. I doubt it causes any measureable perf change.